### PR TITLE
Mask/redact password and secure_json_data in grafana_datasource; require at least puppet 6.1.0

### PR DESCRIPTION
**Pull request by nmaludy**
Previously, when configuring a datasource with a password, the `grafana_datasource` resource would leak on the CLI and in the puppet report:

```shell
Notice: /Stage[main]/Profile::Grafana/Grafana_datasource[postgres]/secure_json_data: secure_json_data changed  to {
  'password' => 'mysweetpassword1'
} (corrective)

```

Now with this fix, the `password` or `secure_json_data` field changes emit the following diff:

```shell
Notice: /Stage[main]/Profile::Grafana/Grafana_datasource[postgres]/secure_json_data: changed [redacted] to [redacted]
```

There were also several linting / whitespace issues caught by CI, i've fixed these in the PR.

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -198,9 +197,9 @@
 
   if $grafana::toml_manage_package and !empty($grafana::ldap_servers) {
     ensure_packages(['toml-pkg'], {
-      ensure   => $grafana::toml_package_ensure,
-      name     => $grafana::toml_package_name,
-      provider => $grafana::toml_package_provider,
+        ensure   => $grafana::toml_package_ensure,
+        name     => $grafana::toml_package_name,
+        provider => $grafana::toml_package_provider,
```
[the remark this answers is not in the document]
Comment (alexjfisher) [reply manifests/install.pp:202]: meh.  I still call it a bug.

**Review by alexjfisher — CHANGES_REQUESTED** [review]
IIRC, this DSL is only available in Puppet 6, so this would break for puppet 5 users??

See https://github.com/puppetlabs/puppetlabs-yumrepo_core/pull/6

**Comment by bastelfreak** [comment]
I guess a puppet 6 requirement is fine now since Puppet 5 is EoL? @nmaludy can you rebase please?

**Comment by nmaludy** [comment]
@bastelfreak rebase complete!

**Comment by bastelfreak** [comment]
can you take a look the linter complains and raise the required puppet version in metadata.json to 6.1.0 ?

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**